manage.py

```diff
@@ -37,6 +37,15 @@ def add_event_identifier():
         save_to_db(event)
 
 
+@manager.command
+def fix_digit_identifier():
+    events = Event.query.all()
```

Comment:
I want to complete this script before my daughter's wedding, so let's not iterate on 15411442... events when there are only 36 events with digit identifier

Reply:
@iamareebjamal I am not sure how I would write a query like that in SqlAlchemy. I need a little help there.

Reply:
`select * from events where identifier ~ '^[0-9\.]+$';`

Convert this to SQLAlchemy